cloudinit/cmd/devel/logs.py

```diff
@@ -27,6 +29,8 @@
     write_file,
 )
 
+LOG = cast(log.CustomLoggerType, logging.getLogger(__name__))
```

Comment:
I don't love using `cast()` - since it can hide real problems, but it seems that mypy doesn't have a better way to introspect custom log levels in the logger singleton from `getLogger()`, so I'm fine with this. Maybe we can find a better way someday.

Reply:
Agreed. I spent a while looking for a better alternative, but I think this is the best we have atm.